src/PrestaShopBundle/Form/Admin/Sell/Address/CustomerAddressType.php

```diff
@@ -106,47 +123,41 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     ]),
                     new ExistingCustomerEmail(),
                 ],
+                'attr' => [
+                    'data-customer-information-url' => $this->router->generate('admin_customer_for_address_information'),
```

Comment:
So this was hardcoded into Twig before 😮 ?

Reply:
Well yeah

 ```
{{ ps.form_group_row(addressForm.customer_email, {'attr': {
            'data-customer-information-url': path('admin_customer_for_address_information') }}, {
            'label': 'Customer email'|trans({}, 'Admin.Orderscustomers.Feature'),
 }) }}

```

Reply:
I meant the URL was generated into Twig, directly, using `path`.
So yes it was 😄